### ISSUE TITLE
Grid not rerendering when isDraggable or isResizable changes

### ISSUE THREAD

**Issue by darekkay** [STRML/react-grid-layout#1151]
Thanks a lot for the update!

However, I am facing issues after upgrading. I have an "Edit" button which toggles between "view mode" and "edit mode". I achieve it in the following way (see [code](https://github.com/darekkay/dashboard/blob/master/src/components/dashboard/index.tsx#L59)):

```jsx
      <ReactGridLayout
        isDraggable={isLayoutEditable}
        isResizable={isLayoutEditable}
        ...
      >
```

After the update, the Grid component does not rerender anymore, making it impossible to enter the "Edit mode". [Here](https://dashboard.darekkay.com/) you can see the edit mode in action (before updating).

I think it is connected to #1123 and might be the same core issue as for #1150. It could also be connected to #892. 

I can create a branch in my repository with the updated version of RGL if it helps you reproducing the issue.

**Comment by motherwaym** [comment]
I'm having issues with onDrop and the placeholder not rendering. I can get the element to render in the DOM if I modify `shouldComponentUpdate` in https://github.com/STRML/react-grid-layout/blob/master/lib/ReactGridLayout.jsx#L201 by adding
```
||  !isEqual(this.state.droppingPosition, nextState.droppingPosition);
```

But it's still invisible. I think it's related to this issue, but I can file a separate issue if necessary.

**Comment by n1ghtmare** [comment]
Hey @darekkay, thanks for reporting this, indeed it seems to perform weird with the new faster comparison. 

@STRML Somehow we missed this, sorry about that. I've run most examples and my own project and didn't notice the issues.

Anyway, I can now reproduce the problems on example-6, adding new items doesn't work unless we compare the children (which I see that you explicitly remove from the comparison for perf reasons).

Also, adding a switch to toggle `isResizable` and/or `isDraggable` on the grid doesn't really work properly (this one I'm not sure why since the props are both in the `fastRGLPropsEqual` comparison, I'm trying to debug it, will shout out if I find anything).

**Comment by STRML** [comment]
Taking a look now.

**Comment by STRML** [comment]
Thanks, fixed in 0.18.2, see https://github.com/STRML/react-grid-layout/blob/master/CHANGELOG.md#0182-feb-26-2020 for more detail.

**Comment by darekkay** [comment]
Thanks for fixing this so fast, now everything works just fine 👍
